internal/cli/connect.go

```diff
@@ -64,13 +64,13 @@ func connectCluster(ctx context.Context, conn config.MachineConnection) (*client
 
 		keyPath := fs.ExpandHomeDir(conn.SSHKeyFile)
 
-		sshConfig := &connector.SSHConnectorConfig{
+		sshCliConfig := &connector.SSHCLIConnectorConfig{
 			User:    user,
 			Host:    host,
 			Port:    port,
 			KeyPath: keyPath,
 		}
-		return client.New(ctx, connector.NewSSHConnector(sshConfig))
+		return client.New(ctx, connector.NewSSHCLIConnector(sshCliConfig))
```

Comment:
We can't switch to `SSHCLIConnector` by default until `SSHCLIConnector.Dialer` is fully supported. Please see another comment https://github.com/psviderski/uncloud/pull/152/files#r2443816679

Reply:
I kinda like it. Right now I've only added it to the config part as `ssh_cli` vs `ssh` types of entries. Done that for testing between the two, but didn't address `--connect` support 😓